src/operator/fusion/fused_op-inl.h

```diff
@@ -543,6 +552,13 @@ __device__ inline DType relu(const DType val) {
   return val > 0 ? val : 0;
 }
 
+__constant__ const float SQRT_2 = 1.4142135623730950488016887242096;
```

Comment:
Are you sure you want to use constant memory here? It should be constexpr or something.

Reply:
Thank you, I just removed "__constant__"